timemachine/fe/single_topology.py

```diff
@@ -309,11 +309,15 @@ def canonicalize_chiral_atom_idxs(idxs):
     return i, jj, kk, ll
 
 
-def setup_end_state(ff, mol_a, mol_b, core, a_to_c, b_to_c):
+def setup_end_state_chiral_potentials(
```

Comment:
This function is poorly named, since it also returns the bond potential as well?

Reply:
Will think of something better

Reply:
Renamed to `setup_end_state_harmonic_bond_and_chiral_potentials` for lack of a better name in https://github.com/proteneer/timemachine/pull/1315/commits/b2d6b0d0354745ca4f66da28c760b287d2487c68